src/libhoney.js

```diff
@@ -118,10 +118,17 @@ export default class Libhoney extends EventEmitter {
   }
 
   _responseCallback(responses) {
-    let queue = this._responseQueue;
-    if (queue.length < this._options.maxResponseQueueSize) {
-      this._responseQueue = this._responseQueue.concat(responses);
+    const [queue, limit] = [
```

Comment:
this is purely because `this._options.maxResponseQueueSize` is long; shortened both for a prettier function call (not married to this)